scripts/test_v2

```diff
@@ -24,6 +24,12 @@ do
       SLOW=--with-slow-tests
       shift # past argument
       ;;
+      --moto-port=*)
+      # prevent port binding issues by explicitly setting the motoserver s3 port
+      # https://developer.apple.com/forums/thread/682332
+      export MOTO_PORT="${key#*=}"
+      shift # past argument=value
+      ;;
```

Comment:
Thank you for adding this, @dmannarino. I'm not sure why I didn't the first time around.... 🤷